packages/samples/headless-react/src/components/result-list/result-link.tsx

```diff
@@ -2,6 +2,14 @@ import {buildInteractiveResult, Result} from '@coveo/headless';
 import {FunctionComponent, useEffect} from 'react';
 import {engine} from '../../engine';
 
+function filterProtocol(uri: string) {
```

Comment:
Copied from the [function in Atomic](https://github.com/coveo/ui-kit/pull/585/files#diff-43593038e126db15f4691e38581d9d284cf146d56de41c43052798171bb7e497R42) by @ThibodeauJF.